backend/experiment/tests/test_views.py

```diff
@@ -190,7 +191,7 @@ def test_serialize_experiment(self):
         )
         self.assertEqual(
             serialized_experiment['image'], {
-                'file': '/upload/test-image.jpg', 'href': '', 'alt': ''}
+                'file': f'{settings.BASE_URL}/upload/test-image.jpg', 'href': '', 'alt': ''}
```

Comment:
You might as well test the serialization of the `href` & `alt` properties here :)